hub/config/src/main/java/uk/gov/ida/hub/config/data/S3ConfigSource.java

```diff
@@ -17,15 +18,23 @@
 import java.io.InputStream;
 import java.text.DateFormat;
 import java.text.SimpleDateFormat;
+import java.util.Date;
 
 public class S3ConfigSource {
     private static final Logger LOG = LoggerFactory.getLogger(S3ConfigSource.class);
     private ConfigConfiguration configConfiguration;
-    private AmazonS3 s3Client;
+    private S3Object s3Object;
```

Comment:
This doesn't look right to me. Does this work? Shouldn't we be getting the s3Object every request.